docs/components/inputMapper.base.md

```diff
@@ -0,0 +1,300 @@
+the `tabbableElements` member variable for use during tab navigation.
+
+### `{inputMapper.base}.stopTrackingDOM()`
+
+- Returns: Nothing.
+
+Stops the `MutationObserver` called by the [`{inputMapper.base}.trackDOM`](#inputmapperbasetrackdomthat) when the
+instance of the `inputMapper.base` component is destroyed. It avoids listening changes to the DOM any further.
+
+## Navigation Invokers
+
+### `{inputMapper.base}.click(value)`
+
+- `value {Number}` Current value of the gamepad input.
+- Returns: Nothing.
+
+Performs **click** on the focused element when the gamepad `button` or `trigger` is pressed. However, the `select`
+dropdown is toggled by changing its `size` attribute when clicked using the gamepad.
+
+### `{inputMapper.base}.previousPageInHistory(that, value)`
+
+- `that {Object}` The `inputMapper.base` component.
+- `value {Number}` Current value of the gamepad input.
+- Returns: Nothing.
+
+Navigates to the previous page in history when the gamepad `button` or `trigger` is pressed. It uses the
+[`History API`](https://developer.mozilla.org/en-US/docs/Web/API/History_API) for the core functionality. The `focused
+element` of current web page is saved using [`chrome.storage`](https://developer.chrome.com/extensions/storage) before
+the operation is performed. It is used for restoring focus on the same element when the user navigates back to the same
+page.
+
+### `{inputMapper.base}.nextPageInHistory(that, value)`
+
+- `that {Object}` The `inputMapper.base` component.
+- `value {Number}` Current value of the gamepad input.
+- Returns: Nothing.
+
+Navigates to the next page in history when the gamepad `button` or `trigger` is pressed. It uses the
+[`History API`](https://developer.mozilla.org/en-US/docs/Web/API/History_API) for the core functionality. The `focused
+element` of current web page is saved using [`chrome.storage`](https://developer.chrome.com/extensions/storage) before
+the operation is performed. It is used for restoring focus on the same element if the user navigates back to the same
+page.
+
+### `{inputMapper.base}.thumbstickHistoryNavigation(that, value, invert)`
+
+- `that {Object}` The `inputMapper.base` component.
+- `value {Number}` Current value of the gamepad input.
+- `invert {Boolean}` Whether the thumbstick history navigation should be in opposite order.
+- Returns: Nothing.
+
+Calls the [`{inputMapper.base}.previousPageInHistory`](#inputmapperbasepreviouspageinhistorythat-value) and
+[`{inputMapper.base}.nextPageInHistory`](#inputmapperbasenextpageinhistorythat-value) invokers according to the
+direction the `thumbstick` is pressed. The direction can be inverted by passing `true` as the `invert` argument.
+
+### `{inputMapper.base}.reverseTab(that, value, direction)`
+
+- `that {Object}` The `inputMapper.base` component.
+- `value {Number}` Current value of the gamepad input.
+- `direction {String}` The direction for backward tab navigation, i.e. `reverseTab`.
```

Comment:
This seems wrong, i.e. the `direction` should be hard-coded in the invoker's definition and shouldn't be accepted as an argument.

Reply:
@the-t-in-rtf it's indeed hard-coded. But removing it raises another question about removing `that` since it's specified/passed in the invoker's `args` option and not when the invoker is called.

Reply:
@the-t-in-rtf thanks for pointing this out. I think I made a mistake with the arguments list here and as well as in the `navigator` documentation as it seems we should only specify those arguments which the user can provide and not the ones used in the invoker's definition.

Reply:
Yes, I didn't even notice that `that` was an argument, but you've got the idea now.